RELEASE-NOTES.rst

```diff
@@ -0,0 +1,76 @@
+..
+    This file is part of RERO MEF.
+    Copyright (C) 2018 RERO.
+
+    RERO MEF is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General Public License as
+    published by the Free Software Foundation; either version 2 of the
+    License, or (at your option) any later version.
+
+    RERO MEF is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with RERO MEF; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, RERO does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+v0.3.0
+======
+
+-  Fixes wrong VIAF PIDs in MEF records.
+-  Updates VIAF, RERO, GND data.
+-  Adds tests to spot duplicated PIDs.
+-  Updates metadata model transformation (from MARC to JSON).
+-  Updates Invenio to version ``3.2.1``.
+-  Integrates IdRef:
+
+   -  Harvest IdRef through OAI-PMH.
```

Comment:
Why is the indentation different here ?

Reply:
Because it's a nested list:

- list:
   - nested list.
- continued list.